cvIgnacioCalahorro.html

```diff
@@ -45,16 +45,10 @@ <h2 class='about'>ABOUT ME:</h2>
         <div id="aboutmostrar" >
             <p class="texto-justificado">
                 I was granted an Erasmus scholarship to work as Intern at the University of Twente, Netherlands to
-                develop a GIS-Tool used in urban planning. I had been working as a Land Surveyor for EPSA International (LLOTOR Arabia) in an open air mine in Saudi Arabia property of Ma'aden Gold. Then I moved to Riyadh (KSA) where I was working for APPLUS+ under the managed of the largest consortium, FAST, as a surveyor throught the Line 5 of the Riyadh Metro Project.
+                develop a GIS-Tool used in urban planning. I had been working as a Land Surveyor for EPSA International (LLOTOR Arabia) in an open air mine in Saudi Arabia property of Ma'aden Gold. Afterwards, I moved to Riyadh (KSA) where I was working for APPLUS+ under the management of the largest consortium, FAST, as a surveyor through the Line 5 of the Riyadh Metro Project.
```

Comment:
- "afterwards" is sexier than "then"
- "managed" is a verb, you can't work under a verb
- "throught" -> "through" is the correct spelling